lib/scenic/adapters/postgres.rb

```diff
@@ -1,5 +1,19 @@
 module Scenic
+  # Scenic database adapters.
+  #
+  # Scenic ships with a Postgres adapter only but can be extended with
+  # additional adapters. The {Adapters::Postgres} adapter provides the
+  # interface.
   module Adapters
+    # An adapter for managing Postgres views.
+    #
+    # **This object is used internally by adapters and the schema dumper and is
+    # not intended to be used by application code. It is documented here for
+    # use by adapter gems.**
```

Comment:
It's tagged with `@api extension` so we can set yard to not include it in generated docs by adding an exclude for it in .yardopts. We haven't done that yet. Probably should.

We can change the implementation here without worry in that case, I think. Changing the interface would depend on whether we know of any extensions.